src/components/TopicSection/TopicButton/index.tsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+import { Button, Typography } from "@material-ui/core";
+
+type Props = {
+  name: string;
+  selected: boolean;
+  handleClick: (topic: string) => void;
```

Comment:
Seems like this type is not correct, should be enough if you set it like this: `handleClick: Function;`.
Doing so, you can remove the `/* eslint-disable no-unused-vars */`.